setup.py

```diff
@@ -24,7 +24,7 @@ def get_file_content(filename):
     license="GPLv2",
     dependency_links=[],
     setup_requires=[],
-    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,<3.9",
+    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,<3.10",
```

Comment:
Is there a reason why 3.10 and upper are excluded? Usually, newer versions of python don't break backward compatibility (Python 2 -> 3 being the exception).

Reply:
I'm not sure if this PR is really the right place to be asking this. We both agree though:

- https://github.com/transifex/transifex-client/pull/303#issuecomment-708437075
- https://github.com/transifex/transifex-client/pull/303#issuecomment-708477512